test/browser.js

```diff
@@ -176,3 +178,56 @@ test('throws if does not support ReadableStream', withPage, async (t, page) => {
 
 	await server.close();
 });
+
+test('FormData with searchParams', withPage, async (t, page) => {
+	const server = await createTestServer();
+	server.get('/', (request, response) => {
+		response.end('nothing');
+	});
+	server.post('/', async (request, response) => {
+		const pBody = util.promisify(body);
+		response.end(await pBody(request));
+	});
+
+	await page.goto(server.url);
+	await page.addScriptTag({path: './umd.js'});
+
+	const requestBody = await page.evaluate(url => {
+		window.ky = window.ky.default;
+		const formData = new window.FormData();
+		formData.append('file', new window.File(['bubblegum pie'], 'my-file'));
+		return window.ky(url, {
+			method: 'post',
+			searchParams: 'foo=1',
+			body: formData
+		}).text();
+	}, server.url);
+
+	t.regex(requestBody, /bubblegum pie/);
+
+	await server.close();
+});
+
+test('headers are preserved when input is a Request and there are searchParams in the options', withPage, async (t, page) => {
+	const server = await createTestServer();
+	server.get('/', (request, response) => {
+		response.end(request.headers['content-type']);
+	});
+
+	await page.goto(server.url);
+	await page.addScriptTag({path: './umd.js'});
+
+	const requestBody = await page.evaluate(url => {
+		window.ky = window.ky.default;
+		const request = new window.Request(url, {
+			headers: {'content-type': 'text/css'}
+		});
+		return window.ky(request, {
+			searchParams: 'foo=1'
```

Comment:
Same here - make sure test fails if search params are missing from the request to the server.

Reply:
done